JavaScript/stock.js

```diff
@@ -0,0 +1,98 @@
+class SearchResult {
```

Comment:
in addition to this class, you should have another one ("searchForm"). The functinalities should be seperated between the clases. Go over the ITC solution when published.